src/handlers/multiSig/multiSigCreated/multiSigCreated.ts

```diff
@@ -26,11 +29,13 @@ export const handleMultiSigMotionCreated: EventHandler = async (
   const multiSigClient = await getMultiSigClient(colonyAddress);
   const oneTxPaymentClient = await getOneTxPaymentClient(colonyAddress);
 
-  const stakedExpenditureClient =
-    await getStakedExpenditureClient(colonyAddress);
+  const stakedExpenditureClient = await getStakedExpenditureClient(
+    colonyAddress,
+  );
 
-  const stagedExpenditureClient =
-    await getStagedExpenditureClient(colonyAddress);
+  const stagedExpenditureClient = await getStagedExpenditureClient(
+    colonyAddress,
+  );
```

Comment:
Looks like these are just on save formatting changes? Maybe we should open an issue to add a .prettierrc file to block-ingestor